doc/dev/crimson/crimson.rst

```diff
@@ -95,36 +96,34 @@ managed by the Seastar framework. They are:
 
 .. describe:: bluestore
 
-   The object store used by classic OSD by default.
+   The object store used by the classic ``ceph-osd``
 
 daemonize
 ---------
 
 Unlike ``ceph-osd``, ``crimson-osd`` does not daemonize itself even if the
-``daemonize`` option is enabled. Because, to read this option, ``crimson-osd``
+``daemonize`` option is enabled. In order to read this option, ``crimson-osd``
 needs to ready its config sharded service, but this sharded service lives
-in the seastar reactor. If we fork a child process and exit the parent after
+in the Seastar reactor. If we fork a child process and exit the parent after
 starting the Seastar engine, that will leave us with a single thread which is
-the replica of the thread calls `fork()`_. This would unnecessarily complicate
-the code, if we would have tackled this problem in crimson.
-
-Since a lot of GNU/Linux distros are using systemd nowadays, which is able to
-daemonize the application, there is no need to daemonize by ourselves. For
-those who are using sysvinit, they can use ``start-stop-daemon`` for daemonizing
-``crimson-osd``. If this is not acceptable, we can whip up a helper utility
-to do the trick.
+a replica of the thread that called `fork()`_. Tackling this problem in Crimson
+would unnecessarily complicate the code.
 
+Since supported GNU/Linux distributions use ``systemd``, which is able to
+daemonize the application, there is no need to daemonize ourselves. 
+Those using sysvinit can use ``start-stop-daemon`` to daemonize ``crimson-osd``.
+If this is not acceptable, that's too damned bad.
```

Comment:
Harsh, but let it stand. I don't really have an informed opinion on the sysvinit/systemd fight.

Reply:
_Crazy People_ reference.  Said fight has clearly been tilting at windmills for years. We had more or less figured out sysvinit, so naturally we had to mess with it.